lib/actions/serverSetupActions.js

```diff
@@ -402,6 +417,33 @@ export function applyServer() {
     };
 }
 
+export function applyServerAgain() {
+    return (dispatch, getState) => {
+        const portToUser = getState().app.adapter.ports
+        .get(getState().app.adapter.selectedAdapterIndex);
```

Comment:
The selected port can be accessed from `getState().core.serialPort.selectedPort`.